packages/flutter_test/test/widget_tester_leaks_test.dart

```diff
@@ -0,0 +1,191 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
+
+import 'utils/leaking_classes.dart';
+
+late final String _test1TrackingOnNoLeaks;
+late final String _test2TrackingOffLeaks;
+late final String _test3TrackingOnLeaks;
+late final String _test4TrackingOnWithCreationStackTrace;
+late final String _test5TrackingOnWithDisposalStackTrace;
+
+void main() {
+  collectedLeaksReporter = (Leaks leaks) => verifyLeaks(leaks);
+  LeakTesting.settings = LeakTesting.settings.copyWith(ignoredLeaks: const IgnoredLeaks(), ignore: false);
+
+  // It is important that the test file starts with group, to test that leaks are collected for all tests after group too.
```

Comment:
[nit] I don't think this comment is true - we also intercept `test` and others, so I would expect any test structure to work fine.

Reply:
True, but we want to test that any structure will work fine. That's why in the test group goes first and then single test, to make sure this story is covered.